codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -23,20 +23,9 @@ def validate(self, input: TermsAgreementInput):
             raise ValidationError("Owner does not have an associated user")
 
     def update_terms_agreement(self, input: TermsAgreementInput):
-        now = timezone.now()
-
-        owner_profile, _ = OwnerProfile.objects.get_or_create(
-            owner=self.current_owner,
-        )
-        owner_profile.terms_agreement = input.terms_agreement
-        owner_profile.terms_agreement_at = now
-        owner_profile.save()
-
-        # Store agreements in user table as well
-        user = User.objects.get(id=self.current_owner.user_id)
-        user.terms_agreement = input.terms_agreement
-        user.terms_agreement_at = now
-        user.save()
+        self.current_user.terms_agreement = input.terms_agreement
```

Comment:
`self.current_user` here is of `User` instance right?

Reply:
Yes